_rules/button-accessible-name-97a4e1.md

```diff
@@ -50,55 +50,55 @@ There are no major accessibility support issues known for this rule.
 
 #### Passed Example 1
 
-Regular button.
+The [accessible name][] of `button` elements comes from their content.
 
 ```html
 <button>My button</button>
 ```
 
 #### Passed Example 2
 
-Value attribute as the [accessible name][].
+The [accessible name][] of the `input` button comes from its `value` attribute.
 
 ```html
 <input type="submit" value="Submit" />
 ```
 
 #### Passed Example 3
 
-`aria-label` for the [accessible name][].
+The [accessible name][] of the `button` comes from its `aria-label` attribute.
 
```

Comment:
There appear to be two different ways of phrasing the example descriptions. One is quite general, such as "The accessible name of button elements comes from their content" or "Elements with role=button require an accessible name. In this example...". The other is specific to the example itself, such as "The accessible name of the input button comes from its value attribute" or "The accessible name of the button comes from its aria-label attribute".

Should we be more consistent about this? Describe in more general terms and then give a code example.

"The accessible name of a button element can come from its content"
"An input element can also be a button. The accessible name of an input button can come from its value attribute."
"The accessible name of a button can come from its aria-label attribute"
"An element with role=button also requires an accessible name. In this example..."

I realise this comment has potentially further reaching implications.

Reply:
I think we definitely should make example descriptions more consistent and have guidelines for them (the same way we reworked the rule names recently). See #834 That need some work agreeing on the guidelines and then even more work going through all rules…

I'd say this is quite independant from this PR, and as long as we do not have the guidelines to follow, it's hard to say that this PR does not comply to these guidelines 😄 

My personal choice for this description would probably be something like
> This `button` element has an [accessible name][] coming from its `aria-label` attribute.

Reply:
The updated guidance in draft PR #946 isn't ready to merge yet. Sorry, my fault for being slow. It currently reads:

> Each test case should be simply and uniquely named, _e.g. "Passed Example 1"_. Each example must also include a brief declarative description that uses plain language, and is a well-formed grammatically correct sentence that finishes with a full stop/period. Words from code should use back ticks and be identified as elements or attributes.
> 
> > _For example:_ "This page has two `title` elements."

Do you think that is clear enough to guide this PR? And to guide #834?